.github/CODEOWNERS

```diff
@@ -0,0 +1,8 @@
+# All files related to dependency management are owned by the
+# currently active dependency manager (DM) to trigger an automatic review
+# request from the DM upon changes. Please see AEP-002 for details:
+# https://github.com/aiidateam/AEP/tree/master/002_dependency_management
+setup.*             @csadorf
+environment.yml     @csadorf
+requirements*.txt   @csadorf
+pyproject.toml      @csadorf
```

Comment:
What is the policy for such specifications concerning the other files of aiida-core?
For which files should we aim to specify code owners?

Reply:
I do not think that such a policy exists yet, but we could of course establish one. Should we that first?

Reply:
I think it would be a good idea to discuss this - at the next AiiDA meeting, and perhaps even before via email; also to explain what the term means in practice for us.
E.g. currently, when assigning reviewers, we've simply been following github's suggestions (based on git blame).

Reply:
The main reason that I added this was to ensure that changes that directly affect the dependency management are reviewed by the DM. I don't think we necessarily have to adopt that for the whole code base.

Reply:
Based on offline discussion, it appears we will not adopt code owners for any other parts of the code.